web-console/4.0-designs/resource-quota/resource-quota.md

```diff
@@ -0,0 +1,48 @@
+# Resource Quota
+
+![](img/resource-quota-details.png)
```

Comment:
nit - how do you get to this page? Would be good to show the nav item highlighted.

Reply:
It's under `Administration -> Resource Quotas -> Quota Details`